spec/lib/export_dsi_user_records_to_big_query_spec.rb

```diff
@@ -42,7 +42,7 @@
       'given_name' => Faker::Name.first_name,
       'family_name' => Faker::Name.last_name,
       'email' => Faker::Internet.email,
-      'organisation' => { 'URN' => 100_000 }
+      'organisation' => { 'URN' => 100_000, 'UID' => 100_000 }
```

Comment:
Please make the values different for the different keys so that the test can fail when the wrong one is returned.